packages/components-js/tests/e2e/puppeteer/specs/scroller.e2e.ts

```diff
@@ -24,19 +24,23 @@ type InitOptions = {
   otherMarkup?: string;
   tag?: 'a' | 'button' | 'span';
   scrollToPosition?: ScrollToPosition;
+  scrollbar?: boolean;
```

Comment:
Would be good to have a more boolean like `hasScrollbar`.

Reply:
Done